web/packages/teleterm/src/ui/TopBar/ConnectMyComputer/ConnectMyComputer.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback, useRef, useState } from 'react';
+import Popover from 'design/Popover';
+import styled from 'styled-components';
+import { Box } from 'design';
+
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { ListItem } from 'teleterm/ui/components/ListItem';
+
+import { ConnectMyComputerIcon } from './ConnectMyComputerIcon';
+
+export function ConnectMyComputer() {
+  const iconRef = useRef();
+  const [isPopoverOpened, setIsPopoverOpened] = useState(false);
+  const appCtx = useAppContext();
+
+  const togglePopover = useCallback(() => {
+    setIsPopoverOpened(wasOpened => !wasOpened);
+  }, []);
+
+  function openSetupDocument(): void {
```

Comment:
We're going to need a function for this in the style of `src/ui/services/workspacesService/documentsService/connectToKube.ts`.

It's not necessary in this PR, it's probably best to wait for the UI to stabilize. But in the future I'd consider moving all of those functions to a file called something like `connectToResources`. Since `connectToConnectMyComputer` doesn't make that much sense, we could just have `openConnectMyComputer`. That function could also automatically handle opening either the setup or the status tab, if we find this to be useful.